app/assets/stylesheets/content/_legacy_actions.sass

```diff
@@ -94,3 +94,14 @@ ul.legacy-actions-more
 
 .message-reply-menu
   @include contextual(-39px)
+
+.action-menu-inline-label
```

Comment:
@ulferts confusingly we're using `.action-menu` for the new (Angular-based) context menu. Some time ago we renamed the classes associated with the older menu to `legacy-actions-*`

also, I haven't documented it yet, but I'm basing my CSS naming on [the ideas expressed in this article](http://www.hagenburger.net/BLOG/Modular-CSS-Class-Names.html) (which in turn are based roughly on the [BEM](https://bem.info/) methodology). In this vein, the class naming would probably end up looking like: `legacy-actions--inline-label`.